src/class_objectProcessor.py

```diff
@@ -1,10 +1,14 @@
+"""
+src/class_objectProcessor.py
+=================================
+"""
 import hashlib
 import random
-import shared
 import threading
 import time
 from binascii import hexlify
 from subprocess import call  # nosec
+import shared
```

Comment:
why?

Reply:
Pylint warning: wrong-import-order. 
Standard import " import threading, import time, from binascii import hexlify, from subprocess import call " should be placed before "import shared"